idaes/gas_solid_contactors/unit_models/moving_bed.py

```diff
@@ -508,55 +516,70 @@ def _make_performance(self):
         gas_phase = self.config.gas_phase_config
         solid_phase = self.config.solid_phase_config
 
+        # Get units meta data from property packages
+        units_meta_gas = \
+            gas_phase.property_package.get_metadata().get_derived_units
+        units_meta_solid = \
+            solid_phase.property_package.get_metadata().get_derived_units
+
         # Declare Mutable Parameters
         self.eps = Param(mutable=True,
                          default=1e-8,
-                         doc='Smoothing Factor for Smooth IF Statements')
+                         doc='Smoothing Factor for Smooth IF Statements',
+                         units=pyunits.dimensionless)
```

Comment:
Be careful here - epsilon needs to have the same units as the quantity it is scaling.

Reply:
Thank you for checking this. This parameter is only used in one place, to smooth the Reynold's number on line 798 and the units match the dimensionless number (which is explicitly given dimensionless units on line 558).